BrainPortal/app/views/tasks/show.html.erb

```diff
@@ -61,29 +61,20 @@
 
     <!-- FAILURE RECOVERY -->
     <% if CbrainTask::FAILED_STATUS.include?(@task.status) %>
-      <%
-        label = (
-          case @task.status
-          when /Setup/       then "Retry Setting Up"
-          when /Cluster/     then "Retry Processing"
-          when /PostProcess/ then "Retry Post Processing"
-          end
-        )
-      %>
-      <%= link_to label, { :action => 'operation', :operation => 'recover', 'tasklist[]' => @task.id }, :class  => "button menu_button", :method => :post %>
+      <%= link_to 'Retry Failed', { :action => 'operation', :operation => 'recover', 'tasklist[]' => @task.id }, :class  => "button menu_button", :method => :post %>
```

Comment:
So what states are allowed? The image is wrong, and it isn't documented anywhere. The "nice labels" don't add anything to the user experience. I'm a bit grumpy because my tasks are all very poorly specified, as are the task states